---
string[] hint_0
string[] hint_1
string[] hint_2
string[] hint_3
string[] hint_4
string[] hint_5
